### DISCUSSION TITLE
[Q] How to use 'numFmt'?

### DISCUSSION THREAD

**Discussion by matdombrock** [exceljs/exceljs#2422] [Q&A]
## 💬 Questions and Help

I have read the documentation concerning this feature but I'm not sure how to actually use this. 

It seems like the documentation here is pretty lacking. Is this some kind of standardized excel feature that I should be able to find documentation on elsewhere (I've looked)? 

What I want to do is take a number like `16.454857719662395` and turn have it be formatted like this:
`16.45%`

I'm attempting to do something like this:
```
style: {
  numFmt: '0.00%',
}
```
But this results in a value like:
```
1645.49%
```

Which is obviously not what I want. It seems like the software is thinking that I'm giving it a rate and asking for it to be converted into a percentage. All I really want to do is supply a number that is a percentage and format it with 2 decimal places and a percent sign at the end. 

Thanks for the help!

**Comment by texas697** [comment]
need same question answered! Except I just need to add a % to the value

**Comment by vincent-ogury** [comment]
You can try directly in excel, then go on "Other format, Custom". You got the string to copy.

To get 16%, use '0_%',
To get 16.45%, use '0.00%'

**Comment by davide75** [comment]
I have a similar issue
sample value => 15.209773191862164
desired result => 15,21%

by using '0.00%' I get this 1521,00

I tried these setup values:

```
style : {
    numFmt: #,##0.00;[Red]-#,##0.00'
}
```
since I want to highlight negative values in red.
Thanks

**Comment by ilianxiliev** [comment]
I had the same issue. 
I wanted to show 15.00% but got 1500.00% by using numFmt = '0.00%' instead.
I resolved my issue by dividing the value I put in this cell by a hundred. In that case it shows it as 15.00%
